connector/connect/pom.xml

```diff
@@ -271,6 +282,31 @@
               <pattern>io.grpc</pattern>
               <shadedPattern>${spark.shade.packageName}.connect.grpc</shadedPattern>
             </relocation>
+
+            <relocation>
+              <pattern>com.google.android</pattern>
```

Comment:
@HyukjinKwon Sorry for the late reply

The relocation rule may be incorrect

I think it should be 

```
<pattern>android.annotation</pattern>
<shadedPattern>${spark.shade.packageName}.connect.android.annotation</shadedPattern>
```

and unzip the `assembly` jar, the contents are as follows:

```
ls

META-INF	android		com		google		grpc		org		spark
```

Not all contents are placed in `${spark.shade.packageName}.connect`, which may require further check

Reply:
Hey sure, please go ahead with a followup. I just rushed to merge to fix up the broken tests with Maven :-).

Reply:
OK~  Maybe do this tomorrow, a little tired today